docs/noise/qcvv/parallel_xeb.ipynb

```diff
@@ -65,7 +65,7 @@
     "    import cirq\n",
```

Comment:
<p>Maybe specify that "RB" means single-qubit randomized benchmarking.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
done